### ISSUE TITLE
Changing the global style, body text and heading color to white, will cause Guternberg editor to show the text in white on top of white background so the text doesn't show

### ISSUE THREAD

**Issue by MosheBiton** [soflyy/oxygen-bugs-and-features#3018]
**Any bug report that doesn't include a Sandbox link or replication steps will be closed. If you can't provide these pieces of information, we're happy to help you via our official support channel instead: https://oxygenbuilder.com/support. You need to select "Something's broken and I need help" from the dropdown box on the left.**

**Describe the bug**
Changing the global style, body text and heading to white (#FFFFFF), will cause Guternberg editor to show the text in white on top of white background so the text doesn't show

*A clear and concise description of what the bug is.*
Gutenberg editor is not usable if the global settings of the text and heading color is set to white.

**A link to a Sandbox site where the bug has been reproduced https://oxygenbuilder.com/try? (REQUIRED)**
*Provide the link to a Sandbox install where the issue is present. If the issue only exists on a specific post or template, provide a direct link to that post or template.*

**IF YOU CAN'T PROVIDE A SANDBOX REPRODUCTION: Steps required to reproduce the bug:**
Open the Oxygen editor and
1. Go to Settings 
2. Click on Global Styles
3. Click on "headings"
4. Change all the colors of the headings to white (#FFFFFF)
5. Go back and click on "body text" and change the color to white (#FFFFFF)
6. Save
7. Create new Post and try to write something, the text will be white on white background, unrecognized.

**If you can't fill out the above information, you should not be filing a bug report. Instead, contact us via https://oxygenbuilder.com/support with the details you have and we'll investigate your issue further.**

**Comment by YVAK** [comment]
My friend, let me get this straight, Your page is white > **_You set_** your text colors to #ffffff; (white) > your text changes to white.


That is not a bug, you have _**set**_ the text to white. If you do not change your background color it will obviously show white on white, am I missing some info or?

**Comment by MosheBiton** [comment]
Hi,

The text in my website is supposed to be white, it's the design... However, it shouldn't affect my ability to use gutenberg... If there was an option to change the background of the gutenberg editor then I could do it but it's not possible.

The text of the website can be white, it's an option, and as long as it's an option, it shouldn't affect the gutenberg editor or any other editor in the backend system.

Right now, my solution is to use javascript to change the background of the page or use an addon to do it for me. I will not change the text color to another one just to use gutenberg as intended, as I want my text color to be white.

**Comment by mentalpiggie1** [comment]
Ahh, now i understand your issue! Is it not possible to target it with some css? Maybe temporarily use inspector to change text colors while editing?

**Comment by MosheBiton** [comment]
Yes, it is possible. I'm using javascript for it (I put a script as a bookmark that does it with a click). There is also the solution of using a plugin that enables dark mode on the backend so I have a way of working and editing. 

It becomes more of an issue when I'm working with other people one creating posts, editing sections on the websites and etc... Each one needs to know what to do in order to use the editor. The dark mode solves it of course but I do think that it's an issue that somehow needs to be addressed.
